src/warnings.cpp

```diff
@@ -47,7 +47,7 @@ std::string GetWarnings(const std::string& strFor)
 
     if (!CLIENT_VERSION_IS_RELEASE) {
         strStatusBar = "This is a pre-release test build - use at your own risk - do not use for mining or merchant applications";
-        strGUI = _("This is a pre-release test build - use at your own risk - do not use for mining or merchant applications");
+        strGUI = _(strStatusBar);
```

Comment:
This doesn't work with translations, imo

Reply:
couldn't this just be `strGUI = strStatusBar`? And `strStatusBar = _(...)`